src/wechaty-mixins/puppet-mixin.ts

```diff
@@ -427,6 +430,95 @@ const puppetMixin = <MixinBase extends WechatifyUserModuleMixin & GErrorMixin &
             })
             break
 
+          case 'tag':
+            puppet.on('tag', async payload => {
```

Comment:
It seems that there has three type events for `Tag`: `tag-create` `tag-remove` and `tag-change`?

Reply:
I thought we use TagEventType do define different kind of tag events?

Reply:
Both are OK, so I should change these three events to the only one `tag` event in `puppet-wxwork`?

Reply:
That's not how we defined tag events in wechaty-puppet? 

https://github.com/juzibot/wechaty-puppet/blob/8538c402ce5ff294cb5b18347b54f18fbe2e0e1e/src/schemas/event.ts#L103-L109

Yes, it is.